server/routes/testutils/appSetup.ts

```diff
@@ -1,20 +1,29 @@
+/* eslint-disable import/first */
```

Comment:
Same issue with importing `applicationInfo.ts` and need to mock first as in `./server/data/testutils/mocks.ts` - see https://github.com/ministryofjustice/hmpps-visits-internal-admin-ui/pull/12#discussion_r1200034777.